openmc/mgxs/mgxs.py

```diff
@@ -214,6 +231,28 @@ def __deepcopy__(self, memo):
         else:
             return existing
 
+    def _add_angle_filters(self, filters):
+        # Add the azimuthal and polar bins if needed.
+        if self.num_polar or self.num_azimuthal:
+            # Then the user has requested angular data, so create the bins
+            if self.num_polar:
+                pol_bins = np.linspace(0., np.pi, num=self.num_polar + 1,
```

Comment:
And if you do make them unity, I don't know that I see a reason to define a `_add_angle_filters(...)` method at all. What was your reasoning?